error/s2n_errno.c

```diff
@@ -239,6 +239,7 @@ static const char *no_such_error = "Internal s2n error";
     ERR_ENTRY(S2N_ERR_UNSUPPORTED_WITH_QUIC, "Functionality not supported when running with QUIC support enabled") \
     ERR_ENTRY(S2N_ERR_PQ_CRYPTO, "An error occurred in a post-quantum crypto function.") \
     ERR_ENTRY(S2N_ERR_PQ_DISABLED, "Post-quantum crypto is disabled.") \
+    ERR_ENTRY(S2N_ERR_DUPLICATE_PSK_IDENTITIES, "The list of pre-shared keys provided contains duplicate psk identities.") \
```

Comment:
I don't think you should specify "pre-shared keys" here. Technically we could throw this error when a pre-shared-key identity is a duplicate of a session resumption identity too. We should never have any duplicate identities of any kind.